index.js

```diff
@@ -0,0 +1,2 @@
+const nombre = prompt('Cual es tu nombre?');
+alert('Hola Usuario!');
```

Comment:
En este caso me parece que seria mejor que no uses interpolacion de strings
```suggestion
alert('Hola' + nombre);
```